chapter_common_operators/conv.md

```diff
@@ -102,29 +110,39 @@ def conv(oc, ic, nh, nw, kh, kw, ph=0, pw=0, sh=1, sw=1):
     oh = conv_out_size(nh, kh, ph, sh)
     ow = conv_out_size(nw, kw, pw, sw)
     # pad X and then compute Y
-    X = tvm.placeholder((ic, nh, nw), name='X')
-    K = tvm.placeholder((oc, ic, kh, kw), name='K')
+    X = tvm.placeholder((ic, nh, nw), name='x')
+    K = tvm.placeholder((oc, ic, kh, kw), name='k')
     PaddedX = padding(X, ph, pw) if ph * pw != 0 else X
     Y = tvm.compute(
         (oc, oh, ow),
         lambda c, i, j: tvm.sum(
             PaddedX[ric, i*sh+rkh, j*sw+rkw] * K[c, ric, rkh, rkw],
-            axis=[ric, rkh, rkw]), name='Y')
+            axis=[ric, rkh, rkw]), name='y')
     return X, K, Y, PaddedX
 ```
 
-Just as what we did in :numref:`ch_vector_add`, we define a function to get the input and output tensors. Again, we fix the random seed so it returns the same results if calling multiple times.
+Just as what we created `get_abc` in :numref:`ch_vector_add`, we define a method to get the input and output tensors. Again, we fix the random seed so it returns the same results if calling multiple times.
 
 ```{.python .input}
 # Save to the d2ltvm package.
-def get_conv_data(oc, ic, n, k, p=0, s=1, generator=None):
+def get_conv_data(oc, ic, n, k, p=0, s=1, constructor=None):
+    """Return random 3-D data tensor, 3-D kernel tenor and empty 3-D output 
+    tensor with the shapes specified by input arguments.
+
+    oc, ic : output and input channels
+    n : input width and height
+    k : kernel width and height
+    p : padding size, default 0
+    s : stride, default 1
+    constructor : user-defined tensor constructor
+    """
     np.random.seed(0)
     data = np.random.normal(size=(ic, n, n)).astype('float32')
     weight = np.random.normal(size=(oc, ic, k, k)).astype('float32')
     on = conv_out_size(n, k, p, s)
     out = np.empty((oc, on, on), dtype='float32')
-    if generator:
-        data, weight, out = (generator(x) for x in [data, weight, out])
+    if constructor is not None:
```

Comment:
any particular reason to change it? `if generator` is shorter

Reply:
It is for being consistent with the `get_abc` method in http://tvm.d2l.ai/chapter_getting_started/vector_add.html#implemeting-with-numpy

BTW, just caught another typo in the title above, will fix it soon as well :)

Reply:
we can remove `is not None`

Reply:
It is also for the consistency issue. Should we remove `is not None` in http://tvm.d2l.ai/chapter_getting_started/vector_add.html#implemeting-with-numpy as well? I am fine with either way.